src/label/Label.ts

```diff
@@ -45,12 +44,6 @@ export const LabelBase = ThemeableMixin(WidgetBase);
 
 @theme(baseCss)
 export default class Label extends LabelBase<LabelProperties>  {
-	@onPropertiesChanged()
-	protected onPropertiesChanged(event: PropertiesChangeEvent<this, LabelProperties>) {
```

Comment:
It **is** actually mandatory in the registry mixin properties; I think that we will need to make this optional for the `label` and handle no registry in the registry mixin (we shouldn't copy `onPropertiesChange` code for registries around the place).